fact_extractor/plugins/unpacking/sevenz/install.sh

```diff
@@ -1,22 +1,29 @@
 #!/usr/bin/env bash
+set -e
 
 cd "$( dirname "${BASH_SOURCE[0]}" )"
 
 echo "------------------------------------"
 echo "      install p7z from source       "
 echo "------------------------------------"
 
-sudo -EH pip3 install --upgrade git+https://github.com/fkie-cad/common_helper_passwords.git
 
 # install newest version of p7zip
 sudo apt-get remove -y p7zip-full
 
-mkdir /tmp/fact_build
+mkdir /tmp/fact_build || :
```

Comment:
Colon only here to send me into a 5 min shell documentation rabbit hole, hm.

Reply:
I googled shell script nop and `:` was the result :sweat_smile:

Reply:
The idea was to not fail if the directory already exists (in case the 7z installation failed halfway through) but I will change it to a `-p` to make it clearer

Reply:
Nah, it's okay. I'm just used to `|| true` for those purposes and was not aware of what `:` does here.